src/elo.py

```diff
@@ -1,3 +1,5 @@
 import numpy as np
 
-print(np.zeros(1))
\ No newline at end of file
+print(np.zeros(1))
+
+print("branchleri deniyorum heyoo")
```

Comment:
Buraya kod recommend edecegim
```suggestion
print(np.zeros([3, 45]))

print("branchleri denedim heyoo")
```